.github/workflows/ci.ubuntu-x64.dev.yml

```
@@ -0,0 +1,54 @@
+    branches: [main, dev]
+  pull_request:
+    branches: [main, dev]
+
+jobs:
+  docker-ci:
+    runs-on: ubuntu-latest
+    name: Docker CI (ubuntu-x64.dev)
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)